specs_e2e/rails_6_1/test.sh

```
@@ -0,0 +1,57 @@
+
+echo '-- migration'
+bundle exec ./bin/rails db:drop || true
+bundle exec ./bin/rails db:create db:migrate
+
+echo '-- cypress install'
+bundle exec ./bin/rails g cypress_on_rails:install --install_folder=test --framework cypress --install_with=npm --force
+rm -vf test/cypress/e2e/rails_examples/using_vcr.cy.js
+
+echo '-- start rails server'
+# make sure the server is not running
+(kill -9 `cat ../server.pid` || true )
```

Shellcheck (0.10.0)

[warning] Quote this to prevent word splitting.

(SC2046)